tests/integration_tests/run_constant_reprocessing_openmc/materials.xml

```diff
@@ -1,34 +1,28 @@
 <?xml version='1.0' encoding='utf-8'?>
 <materials>
-  <material depletable="true" id="1" name="fuel" volume="48710000.0">
-    <density units="g/cm3" value="3.35" />
-    <nuclide name="Li7" wo="0.0787474673879085" />
-    <nuclide name="Be9" wo="0.0225566879138321" />
-    <nuclide name="F19" wo="0.454003012179284" />
-    <nuclide name="Th232" wo="0.435579130482336" />
-    <nuclide name="U233" wo="0.00911370203663893" />
+  <material depletable="true" id="1" name="fuel" volume="0.5541769440932395">
```

Comment:
How come this was changed from MSR to LWR test?

Reply:
the MSBR model takes a very long time to run, so I changed it to a PWR pincell model